packages/experimental/Badge/src/Badge.tsx

```diff
@@ -39,20 +39,24 @@ export const Badge = compose<BadgeType>({
     const Slots = useSlots(userProps, (layer) => badgeLookup(layer, userProps));
 
     return (final: BadgeProps, ...children: ReactNode[]) => {
-      const { icon, iconPosition = 'before', ...mergedProps } = mergeProps(badge, final);
+      const { icon, iconPosition = 'before', size, ...mergedProps } = mergeProps(badge, final);
```

Comment:
nit: setting the default of iconPosition to before seems better to do in useBadge, so that all of the settings of props are in one place.

Reply:
Oh, I see it's already in useBadge. I don't know why I duplicated it here. Thank you for catching this